src/components/scene/vr-mode-ui.js

```diff
@@ -71,6 +60,8 @@ module.exports.Component = registerComponent('vr-mode-ui', {
 
     this.orientationModalEl = createOrientationModal(this.exitVR);
     this.el.appendChild(this.orientationModalEl);
+
+    this.onVRModeChange();
```

Comment:
What about calling this method something like 'updateEnterVRInterface' instead of being bound to the event name is attached to?